DOCS-12525 [EDITS]Copy to edit (#180)
mulesoft/docs-mule-sdk: modules/ROOT/pages/rest-sdk/edits/features.adoc

> = Overview

Reviewer: Apologies: such a short title that I edited out of order!
Again, "Overview" is a super generic title: overview of what? all of REST SDK? something specific about REST SDK? Do we mean How REST SDK Works?
I think what you want her is what the .adoc title implies: features. And features that are different from REST Connect.
So...maybe something like...:
```suggestion
= REST SDK Features
```
or
```suggestion
= REST SDK Metadata-Driven Features
```
or
```suggestion
= REST SDK Feature Improvements Over Rest Connect
```
or....  Whatever your main focus is.

ifndef::env-site,env-github[]
include::../_attributes.adoc[]
endif::[]

REST SDK provides metadata-driven features that create a unified connectivity
experience for all types of users:

* As an API developer, use REST SDK to produce connectivity at scale.
* As an integration specialist, use REST SDK to create connections when
your desired connector is not available in Studio.
* As an API product manager, use REST SDK to expand API outreach to developers

> and non-developer users.

Reviewer: Per style ref., hyphenation:
```suggestion
and nondeveloper users.
```

> * As an API enthusiast, use REST SDK to develop and publish use-case based

Reviewer: ```suggestion
* As an API enthusiast, use REST SDK to develop and publish use case-based
```

connections for a growing marketplace.

> 

Reviewer: This is an awesome list!

Author: IKR :)

== Features in REST SDK

REST SDK currently supports the following features:

* Instant connector generation from RAML 1.0, OAS 2.0, and OAS 3.0

> * Maven distribution framework

Reviewer: ```suggestion
* Apache Maven distribution framework
```
This is just something I know: The Apache Software Foundation tends to be a bit sticky about use of its trademarks, so it's safest to provide attribution of them when introducing them in text. If you mention Maven a lot, it's OK to do so without Apache, once that has been established.

Author: Good to know!

* Paginated operations
* Triggers
* Value providers
* Sample data
* Security scheme support for:
** Basic authentication
** Bearer authentication
** Digest authentication
** Pass-through authentication
** OAuth 2.0 client credentials grant type
** OAuth 2.0 authorization code grant type
* TLS support
* Specific API operation filtering
* Custom headers and query parameters at the global and operation levels
* Global and operation-specific timeouts
* Support for reconnection
* HTTP proxy support
* Automatic error handling
* Support for multiple base URLs

> * Ability to modify or extend the generated connector using Java SDK. You can:

Reviewer: ```suggestion
* Provision of the ability to modify or extend the generated connector using Java SDK in the following ways:
```

** Add additional security schemes
** Add dynamic DataSense
** Add message sources or expression functions
** Customize operations
** Describe custom authentication

> On the non-functional side, REST SDK connectors are fully

Reviewer: ```suggestion
On the nonfunctional side, REST SDK connectors are fully
```

> xref:mule-sdk/non-blocking-operations[non-blocking] and also provide reduced

Reviewer: ```suggestion
xref:mule-sdk/non-blocking-operations[nonblocking] and also provide reduced
```

and optimized extension models.

== Differences with Rest Connect

> The main differences between REST SDK and Rest Connect v1 are:

Reviewer: If the "v1" matters (like there are other versions to which REST SDK is not different?), it should be in the title too; if not, I'd omit globally.

> * Unlike Rest Connect v1 which generates and deploys an unmodifiable

Reviewer: ```suggestion
* Unlike Rest Connect v1, which generates and deploys an unmodifiable
```

connector, REST SDK generates source code that you can version, modify,
and extend manually.
* REST SDK does not publish the generated connectors automatically. You
can deploy the connector at your convenience.
* REST SDK generated source code uses Java SDK instead of XML source
code.

== What's Next

> Now that you've gotten an understanding of how REST SDK works and its provided

Reviewer: See prev. edit comment about "now" and "proceed." Your choice, however!
